hack/make-rules/test-cmd-util.sh

```diff
@@ -526,6 +531,7 @@ run_pod_tests() {
   kubectl delete configmap test-configmap --namespace=test-kubectl-describe-pod
   kubectl delete pdb/test-pdb-1 pdb/test-pdb-2 pdb/test-pdb-3 pdb/test-pdb-4 --namespace=test-kubectl-describe-pod
   kubectl delete namespace test-kubectl-describe-pod
+  kubectl delete statefulset test-sts --namespace=test-kubectl-describe-pod
 
```

Comment:
seems you delete this after delete namespace.

Reply:
Aha. Thanks for pointing out.